error.vue

```diff
@@ -84,15 +84,12 @@
   </div>
 </template>
 
-<script setup>
-const props = defineProps({
-  error: {
-    default: undefined,
-    type: Object || undefined,
-  },
-});
+<script setup lang="ts">
+import { NuxtError } from '#app';
 
-useThemeHead();
+defineProps({
+  error: NuxtError,
+});
```

Comment:
I couldn't use `defineProps<>` or do `import type { NuxtError }` using the current ESLint setup